### ISSUE TITLE
Allow changing the cursor symbol when using zellij

### ISSUE THREAD

**Issue by matteematt** [zellij-org/zellij#484]
I have my vim configured to change the cursor symbol when using different vim modes. However the configuration I currently have does not work in Zellij. Do I need to have some different config, or would this be a feature request?

Here is the config I usually use:
```vim
if g:os == "Darwin"
  " MacOS
  " Tested working in iterm2
  let &t_SI.="\e[5 q" "SI = INSERT mode
  let &t_SR.="\e[4 q" "SR = REPLACE mode
  let &t_EI.="\e[1 q" "EI = NORMAL mode (ELSE)
elseif g:os == "Linux"
  " works in Alacritty on Arch Linux
  let &t_SI = "\<Esc>[6 q"
  let &t_SR = "\<Esc>[4 q"
  let &t_EI = "\<Esc>[2 q"
endif
```

Here is an example of what I expect (using MacOS and iterm2):
Insert mode:
![Screenshot 2021-05-11 at 08 59 28](https://user-images.githubusercontent.com/43502076/117780584-4c67a500-b22f-11eb-84f8-de4296d44d5a.png)
Normal mode:
![Screenshot 2021-05-11 at 08 59 35](https://user-images.githubusercontent.com/43502076/117780626-55587680-b22f-11eb-8c3a-9a173b2279d0.png)

Here is what I am seeing in ZelliJ (also using MacOS and iterm2):
Insert mode (notice the block cursor):
![Screenshot 2021-05-11 at 08 58 25](https://user-images.githubusercontent.com/43502076/117780713-699c7380-b22f-11eb-9f33-b6d37806a564.png)
Normal mode:
![Screenshot 2021-05-11 at 08 59 35](https://user-images.githubusercontent.com/43502076/117780731-6dc89100-b22f-11eb-9fb2-4c28933bf31d.png)

**Basic information**

`zellij --version`: zellij 0.7.0
`tput lines`: 60
`tput cols`:282
`uname -av` or `ver`(Windows): Darwin UKC02YDAXPJG5H 20.4.0 Darwin Kernel Version 20.4.0: Thu Apr 22 21:46:47 PDT 2021; root:xnu-7195.101.2~1/RELEASE_X86_64 x86_64

`alacritty --version`: alacritty 0.7.2 (5ac8060)
`vim --version`:
```
VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Dec 19 2020 21:23:05)
macOS version
Included patches: 1-2164
Compiled by root@Mac-1608412769109.local
Huge version with MacVim GUI. 
```

I have also tried Iterm2 and I am seeing the same error: Build 3.4.6

**Comment by imsnif** [comment]
This should be fixed in `0.9.0`

**Comment by matteematt** [comment]
I have just updated to the latest version and the cursor is indeed working as I expect, thanks
